### PR TITLE
Yordis/refactoring

### PR DESCRIPTION
**Pull request by yordis**
- Fix the max width on multiple files
- Fix links
- Improve default environment for testing

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 Copyright 2019 Jonathan Clem
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```
Comment (yordis) [inline LICENSE.md]: Just fix the max width

File: test/Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ LABEL "com.github.actions.color"="orange"
 
 ARG MIX_HOME=/.mix
 ENV MIX_HOME=$MIX_HOME
-ARG MIX_ENV=dev
+ARG MIX_ENV=test
```
Comment (yordis) [inline test/Dockerfile]: This is a better default for this image, no?!

File: README.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 
-An example workflow to run `mix test` and `mix format --check-formatted` is as follows:
+An example workflow to run `mix test` and `mix format --check-formatted` is as
+follows:
 
 ```hcl
 workflow "Tests & Formatting" {
     on = "push"
-    resolves = ["Test", "Check Formatting"]
+    resolves = ["Run Tests", "Check Formatting"]
```
Comment (yordis) [inline README.md:16]: The other examples are using this.

**Comment by acrogenesis** [comment]
Nice

**Comment by yordis** [comment]
@jclem ping 🙂

**Comment by jclem** [comment]
Sorry for the delay! Missed it in my deluge of notifications. I should be able to take care of this tomorrow or early in the week.

**Review by jclem — APPROVED** [review]

**Comment by jclem** [comment]
Thanks @yordis! Sorry this took so long.

**Pull request merged**